licenses.d2iq.yaml

```diff
@@ -334,22 +334,22 @@ resources:
       - license_path: LICENSE
         ref: ${image_tag}
         url: https://github.com/brancz/kube-rbac-proxy
-  - container_image: ghcr.io/fluxcd/helm-controller:v0.37.4
+  - container_image: ghcr.io/fluxcd/helm-controller:v1.0.1
```

Comment:
1.1.0 is latest. any reason we want to stick to May release?

Reply:
same as other comments, trying to keep things in scope given limited time left for 2.13 release.